addons/crm/models/crm_lead.py

```diff
@@ -1899,6 +1899,11 @@ def _prepare_customer_values(self, partner_name, is_company=False, parent_id=Fal
     def _creation_subtype(self):
         return self.env.ref('crm.mt_lead_create')
 
+    def _creation_subtype_message(self):
+        self.ensure_one()
+        return _('A new lead has been created for the team "%(team_name)s".',
```

Comment:
"for the team ." if no team ? Maybe we could have an explicit management like "A new lead has been created and is not assigned to any team", what do you think ?